src/modules/qt/filter_gpstext.cpp

```diff
@@ -252,7 +254,31 @@ static void gps_point_to_output(mlt_filter filter,
             snprintf(gps_text, 4, "%s", bearing_to_compass(raw.bearing));
         } else
             snprintf(gps_text, 4, "%s", bearing_to_compass(crt_point.bearing));
-    } else if (!strncmp(keyword, "gps_vdist_up", strlen("gps_vdist_up"))
+    }
+	else if (!strncmp(keyword, "gps_cadence", strlen("gps_cadence")) && crt_point.cad!=GPS_UNINIT) {
+		if (strstr(keyword, "RAW")) {
```

Comment:
This is formatted badly. We do not use tabs any more for indent. We never really wanted expressions crammed together (spaces before and after operators). Please reformat it using `clang-format` version 14.